tests/Service/BreadcrumbTest.php

```diff
@@ -25,15 +25,16 @@ public function testBreadcrumbGeneration()
         $breadcrumb = new Breadcrumb(
             new BreadcrumbTestTranslator(),
             new BreadcrumbTestUrlGenerator(),
-            new State(null, [
+            new State(\null, [
                 'space_id' => 'spaceId',
                 'delivery_token' => 'deliveryToken',
                 'preview_token' => 'previewToken',
             ], 'locale')
         );
 
         $breadcrumb->add('item1', 'route1', ['param1' => 'value1'])
-            ->add('item2', 'route2', [], false);
+            ->add('item2', 'route2', [], \false)
+        ;
```

Comment:
this newline looks out of place.

Reply:
That's correct, the coding standard setting for chained, multiline method calls requires the semicolon to go on a new line 🙂